strings.txt

```diff
@@ -9530,7 +9547,7 @@ SETUP_SN_REPORT_STATS_ENABLE
 	FI	Kyllä, auta parantamaan mysqueezebox.com-verkkoa raportoimalla tilastot.
 	FR	Autoriser la collecte d'informations pour contribuer à l'amélioration de mysqueezebox.com.
 	IT	Sì, autorizzo la raccolta dei dati a fini statistici per migliorare mysqueezebox.com.
-	NL	Ja, statistieken rapporteren om mysqueezebox.com te verbeteren.
+	NL	Ja, statistieken rapporteren.
```

Comment:
Please leave the information about mysqueezebox.com in. The user needs to know what the data would be collected for.